src/main/java/com/greghaskins/spectrum/model/BlockConfiguration.java

```diff
@@ -9,13 +9,13 @@
 import java.util.Set;
 
 /**
- * Pre conditions and configurations that apply to a {@link ConfiguredBlock}.
+ * Preconditions and configurations that apply to a {@link ConfiguredBlock}.
  */
 public class BlockConfiguration {
   /**
    * Contains factory methods for creating {@link BlockConfiguration} objects so you can include
    * them in {@link ConfiguredBlock#with(BlockConfiguration, Block)}. The
-   * {@link BlockConfiguration} object has fluent setters so you can add more properties.
+   * {@code BlockConfiguration} object has fluent setters so you can add more properties.
```

Comment:
`Style` `Minor`

Only hyperlink the first occurrence of an artifact in a section. Too many hyperlinks makes a block look messy and (arguably) harder to read.